framework.go

```diff
@@ -65,7 +68,21 @@ func handleHandler(h RouteHandler) http.HandlerFunc {
 			routeURL := url.URL{Path: r.URL.Path, RawQuery: r.URL.RawQuery}
 			w.Header().Set("Link", resp.Pages().GetLinks(routeURL.String()))
 		}
-
+		if resp.Status() >= http.StatusInternalServerError {
+			m := message.Fields{
+				"message": "request encountered internal error",
+				"method":  r.Method,
+				"remote":  r.RemoteAddr,
+				"request": GetRequestID(ctx),
+				"path":    r.URL.Path,
+				"status":  resp.Status(),
+				"error":   resp.Data(),
+			}
+			if len(r.URL.Query()) > 0 {
+				m["params"] = r.URL.Query()
+			}
+			grip.Error(m)
```

Comment:
Could we log this at a debug level instead? I'm not sure we want this affecting our error rate.

Reply:
Also not sure if we can put this finishLogger where we do the original logging, we might not have the right information

Reply:
Changed to debug. I wanted to put the change in `finishLogger` initially as well but its ResponseWriter param doesn't have a way currently to retrieve an error message directly. We could change the negroni ResponseWriter implementation to intercept the error message in a buffer, store it, and then pass it on to the ResponseWriter but that seemed to me more trouble than it's worth.